lib/install/validate/config.go

```diff
@@ -557,3 +560,52 @@ func (v *Validator) CheckDrs(ctx context.Context) {
 	log.Info("DRS check OK on:")
 	log.Infof("  %q", v.Session.Cluster.InventoryPath)
 }
+
+// check that PersistNetworkBacking is set
+func (v *Validator) CheckPersistNetworkBacking(ctx context.Context) {
+	errMsg := "vCenter settings check SKIPPED"
+	if !v.sessionValid(errMsg) {
+		return
+	}
+	if !v.IsVC() {
+		log.Info(errMsg)
```

Comment:
use `op := trace.FromContext(ctx, "Check vCenter serial port backing")` or similar and then log using `op.Infof`.
This will ease integration with the UI feature branch

Reply:
fixed